dmdevtools/invoke_tasks.py

```diff
@@ -93,6 +93,10 @@ def npm_install(c):
 @task(npm_install)
 def frontend_build(c, gulp_environment=""):
     """Build frontend assets using node.js"""
+    if not Path("gulpfile.js").exists:
+        c.run("npm run build")
+        return
```

Comment:
Why do we need to do this?

Reply:
Oh, I didn't mean to commit that haha

It's in case https://github.com/alphagov/digitalmarketplace-user-frontend/pull/316 gets merged...

But it definitely doesn't need to be in this commit, I will rebase it out, thanks for catching it.